apps/system/js/notifications.js

```diff
@@ -228,6 +229,7 @@ var NotificationScreen = {
     this._touchStartX = evt.touches[0].pageX;
     this._touchStartY = evt.touches[0].pageY;
     this._touchPosX = 0;
+    this._touchStartDate = Date.now();
```

Comment:
ha! I think I made the change then trashed it at some point while working on something else... sorry :)